### PR TITLE
Make the store email clickable in the footer

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The store email is not clickable in the footer
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-3181
| How to test?  | Check if the store email can be clickable in the FO footer.

### CODE REVIEW

**Comment by xBorderie** [comment]
This introduces HTML tags in a translatable strings, which we are trying to prevent (hence the [1][/1] pseudp-tags, which are replaced by <span></span>). So, isn't there a solution which would avoid introducing HTML tags?

ping @AlexEven

**Comment by vincentbz** [comment]
Hi @fatmaBouchekoua 

Could you move it to 1.7.2.x branch please ?
So that it can be taken in 1.7.2.1 version.

Thanks !

File: themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl [hunk 1]
```diff
@@ -56,7 +56,7 @@
         <br>
         {* [1][/1] is for a HTML tag. *}
         {l
-          s='Email us: [1]%email%[/1]'
+          s='Email us: <a href="mailto:%email%" class="dropdown">[1]%email%[/1]</a>'
```
Comment (maximebiloe) [inline themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl]: No HTML tags into translated strings.
Please, make the same as the `<span>` tag.

**Review by maximebiloe — CHANGES_REQUESTED** [review]

File: themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl (hunk 1 above)
Comment (fatmaBouchekoua) [reply themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl]: Done. Thanks!

File: themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl [hunk 2]
```diff
@@ -56,10 +56,12 @@
         <br>
         {* [1][/1] is for a HTML tag. *}
         {l
-          s='Email us: [1]%email%[/1]'
+          s='Email us: [1][2]%email%[/2][/1]'
           sprintf=[
-            '[1]' => '<span>',
-            '[/1]' => '</span>',
+            '[1]' => '<a href="mailto:%email%" class="dropdown">',
```
Comment (maximebiloe) [inline themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl]: And, how the %email% will be replaced? 🤔

**Review by maximebiloe — DISMISSED** [review]

File: themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl (hunk 2 above)
Comment (aleeks) [reply themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl]: email

Comment (aleeks) [reply themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl]: nope

**Comment by aleeks** [comment]
done ;)

email

**Comment by fatmaBouchekoua** [comment]
Thank you @aleeks

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thanks @fatmaBouchekoua

**Pull request merged**